spread.yaml

```diff
@@ -1,24 +1,115 @@
 project: ubuntu-image
 
+environment:
+    # On some distros the default GOPROXY setting is 'direct' (eg. Fedora).
+    # Some of the external go packages may get removed or could be temporary
+    # offline (as happened with maze.io/x/crypt), and then the sources are
+    # only available through the proxy cache. Play it safe and enable the
+    # proxy to allow for least CI interruptions.
+    GOPROXY: https://proxy.golang.org,direct
+    LANG: "C.UTF-8"
+    LANGUAGE: "en"
+
 backends:
-  lxd:
+  google:
+    key: '$(HOST: echo "$SPREAD_GOOGLE_KEY")'
+    location: snapd-spread/us-east1-b
+    halt-timeout: 2h
+    systems:
+      - ubuntu-16.04-64:
+            workers: 8
```

Comment:
I think you can use "workers: 1" or "workers: 2"
In snapd we use 8 becase the number of tests is about 500 per system